pkg/hyperkube/hyperkube.go

```diff
@@ -164,6 +165,9 @@ func (hk *HyperKube) Run(args []string, stopCh <-chan struct{}) error {
 
 	s, err := hk.FindServer(serverName)
 	if err != nil {
+		if len(args) > 0 {
+			goto RunAgain // let's keep trying.
```

Comment:
nit: May want to clarify that what we are really doing is chopping off the left-most os arg and trying with the remaining args to deal with varying entry points.